aws/resource_aws_acm_certificate.go

```diff
@@ -149,7 +210,11 @@ func resourceAwsAcmCertificateRead(d *schema.ResourceData, meta interface{}) err
 		if err := d.Set("validation_emails", emailValidationOptions); err != nil {
 			return resource.NonRetryableError(err)
 		}
-		d.Set("validation_method", resourceAwsAcmCertificateGuessValidationMethod(domainValidationOptions, emailValidationOptions))
+
+		if _, ok := d.GetOk("validation_method"); ok {
```

Comment:
Found during testing: wound up switching `validation_method` to `Computed: true` (for now) instead of using this, since it breaks Terraform resource import functionality for this attribute.